.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
```

Comment:
Can you please briefly explain what have you done here, so that I can get a better idea about what your contribution will do?
Also, if possible, please attach screenshots if you have tested this.

Reply:
I want add eslint into project. Contributor can check it in locally
Do you want i run eslint in locally then screenshots it?

Reply:
![image](https://user-images.githubusercontent.com/32766920/95646465-fe50be80-0af2-11eb-9508-be18c83ae528.png)

![image](https://user-images.githubusercontent.com/32766920/95646473-0d377100-0af3-11eb-9599-fc386d5b64e1.png)

Reply:
Yes please, it will be helpful.
Also, can you add a check which points out what files were changed in PR.

Reply:
Yes, this looks good, will add it!

Reply:
You mean to fix problems of eslint and push in this pull request

Reply:
Yes, I will merge this pull request.
Can you also add checks on pull request which check if the new code does not break the previous code? Then, I will open a new issue.

Reply:
Thanks. You can adding the "hacktoberfest-accepted" label to the PR